.github/workflows/eventlist.yml

```diff
@@ -45,6 +45,13 @@ jobs:
         run: mkdir build
         working-directory: ./tools/eventlist
 
+      - name: Generate windows resources
+        if: github.event_name == 'release'

```

Comment:
```suggestion
```

Reply:
Addresed in https://github.com/ARM-software/CMSIS-View/pull/4/commits/b3dadd31d38d3910f228f20c3fe29964969575d4